components/dashboard/src/components/PillLabel.tsx

```diff
@@ -8,7 +8,7 @@ export type PillType = "info" | "warn" | "success" | "neutral";
 
 const PillClsMap: Record<PillType, string> = {
     info: "bg-blue-50 text-blue-500 dark:bg-blue-500 dark:text-blue-100",
-    warn: "bg-orange-100 text-orange-700 dark:bg-orange-600 dark:text-orange-100",
```

Comment:
where doe we use this?

Reply:
[`Enterprise` Pill label](https://github.com/gitpod-io/gitpod/pull/19788#issuecomment-2136667852) is coming from here only. Additionally, it is also being used in Access tokens page (unrelated to this change)